personalpalate/orm/model.py

```diff
@@ -1,13 +1,46 @@
+from datetime import date
 from typing import Optional
 
 from sqlmodel import SQLModel, Field
 from sqlalchemy import PrimaryKeyConstraint
 
 
-# TODO add model
-class AccountDTO(SQLModel):
+class AccountDTO(SQLModel, table=True):
     email: str = Field(primary_key=True)
+    followable: bool
 
-
-class Account(AccountDTO, table=True):  # Subclass to avoid sending password to client
+class Account(AccountDTO):  # Subclass to avoid sending password to client
     password: str
+
+class Follow(SQLModel, table=True):
+    email: str = Field(default=None, foreign_key="account.email")
+    followingEmail: str = Field(default=None, foreign_key="account.email")
+
+    __table_args__ = (
+        PrimaryKeyConstraint("email", "followingEmail"),
+    )
+
+class Meal(SQLModel, table=True):
+    mealID: Optional[int] = Field(default=None, primary_key=True)
+    email: str = Field(default=None, foreign_key="account.email")
+    mealName: str
+    category: str
+    description: str
+    mealFrequency: int
+    dateMade: date
+
+class Ingredients(SQLModel, table=True):
+    ingredientID: Optional[int] = Field(default=None, primary_key=True)
+    mealID: int = Field(default=None, foreign_key="meal.mealID")
+    ingredient: str
+    quantity: str
```

Comment:
We need two feilds, one for quantity and one for unit. Where quantity is a float